components/RHPContainer.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import {Pressable, View, Dimensions} from 'react-native';
+const isSmallScreenWidth = Dimensions.get('window').width <= 800;
+
+const RHPContainer = ({navigation, children, contentStyle}) => {
+    return (
+        <View style={{flexDirection: 'row', width: '100%', height: '100%'}}>
+            {!isSmallScreenWidth && <Pressable style={{ flex: 1 }} onPress={() => navigation.goBack()}>
```

Comment:
Same comments from the other PR. Still not convinced we should use the custom navigator for small screens at all.